pyhub/mcptools/microsoft/excel/com/pool.py

```
@@ -0,0 +1,295 @@
+                    else:
+                        # Wait a bit and retry
+                        time.sleep(0.1)
+
+        if instance is None:
+            raise TimeoutError(f"Could not get Excel instance within {timeout} seconds")
+
+        instance.mark_used()
+
+        try:
+            yield instance.app
+        except Exception as e:
```

Ruff (0.11.9)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)